setup/rpi.md

```diff
@@ -116,13 +116,32 @@ catkin_make
 source devel/setup.bash
 ```
 
-The rover has some customizable settings that will overwrite the default values. Whether you have any changes compared to the defaults or not, you have to manually create these files:
+The rover has some customizable settings that will overwrite the default values. 
+Whether you have any changes compared to the defaults or not, you have to manually create these files:
 ```
 cd ~/osr_ws/src/osr-rover-code/ROS/osr_bringup/config
 touch physical_properties_mod.yaml roboclaw_params_mod.yaml
 ```
-To change any values from the default, modify these files instead so they don't get tracked by git. The files follow the same structure as the default.
 
+To change any values from the default, modify these files instead so they don't get tracked by git. 
+The files follow the same structure as the default. Just include the values that you need to change as the default
+values for other parameters may change over time.
+
+You might also want to modify the file `ROS/osr_bringup/osr.launch` to change the velocities the gamepad controller will
+send to the rover. These values in the node joy2twist are of interest:
+```
+<param name="scale_linear" value="0.18"/>
+<param name="scale_angular" value="0.4"/>
+<param name="scale_linear_turbo" value="0.24"/>
+```
+The maximum speed your rover can go is determined by the no-load speed of your drive motors. The default value is located
+in the file [physical_properties.yaml](ROS/osr_bringupconfig/physical_properties.yaml) as `drive_no_load_rpm`. 
+Then the value of `scale_linear_turbo` can be calculated as `drive_no_load_rpm * 2pi / 60 * wheel radius (=0.075m)`.
+Based on this max value, let's set our regular moving speed to a fraction of that which you can configure to your liking.
```

Comment:
I don't understand this. You say it can be calculated above, but then you say it's a max and its actual value should be 0.75 times itself. Update wording to clarify that the first calc is the max?

Reply:
Update looks good.